.github/.wordlist.txt

```diff
@@ -486,8 +486,11 @@ IPv
 ISCAN
 itemName
 iterable
+jdk
 jinja
 jlink
+jre
+JDK
```

Comment:
nit: I believe wordlist may be case insensitive, in which case JDK and jdk are not both needed.